Packages/install_dependencies

```diff
@@ -21,22 +21,11 @@ python-virtualenv \
 swig \
 socat \
 libyaml-dev \
-libcogl20 \
-libclutter-1.0-0 \
-libclutter-imcontext-0.1-0 \
-libcluttergesture-0.0.2-0 \
 python-scipy \
+python-numpy \
```

Comment:
Yes, needed. Numpy takes a very long time to recompile when loaded by pip.

Reply:
It's still there in install_redeem.